client/pages/form-pages/authorize.tsx

```diff
@@ -0,0 +1,58 @@
+import { Form, Formik, FormikProps } from 'formik';
+import { Field, FormStepTitles, FormSteps } from '@components';
+import { AuthorizationInterface } from 'constants/interfaces';
+
+const initialValues = {
+  projectManagerSignature: '',
+  financialOfficerSignature: '',
+};
+ 
+export const AuthorizationPage: React.FC = () => {
+  return (
+    <Formik initialValues={initialValues} onSubmit={() => {}}>
+      {({ values }: FormikProps<AuthorizationInterface>) => (
+        <Form className='flex justify-center'>
+          <div className='w-2/4 p-4 gap-y-6 bg-white flex flex-col items-center'>
+            <div className='mb-4 flex items-center flex-col'>
+              <h1 className='text-xl font-medium text-bcBluePrimary'>
+                {FormStepTitles[FormSteps.AUTHORIZATION]}
+              </h1>
+              <h3>Please proivde detailed information for your application.</h3>
+            </div>
+
+            <p className='font-bold'>
+              We certify that the information contained in this application, is to the best of our
+              knowledge correct, complete and has been submitted with the concurrence of our Council
+              or Board.
+            </p>
+
+            <Field
+              name='projectManagerSignature'
+              type='text'
+              label='Project Manager, Name and Signature'
```

Comment:
Signature should be a separate field. Use something like this for signature https://www.npmjs.com/package/react-signature-canvas

Reply:
Added signature-canvas package. Looks great , thanks.
![image](https://user-images.githubusercontent.com/13221550/199253502-e6c820e8-ec44-46b1-b705-57b7e37a5ff5.png)